lib/connextclient/ConnextClient.ts

```diff
@@ -464,18 +465,20 @@ class ConnextClient extends SwapClient {
     if (this._reconcileDepositSubscriber) {
       this._reconcileDepositSubscriber.unsubscribe();
     }
-    // TODO: increase interval after testing to 30+ sec
-    this._reconcileDepositSubscriber = interval(5000)
+    const ethBalance$ = interval(30000).pipe(
```

Comment:
I think it may be helpful in the future to comment what this is doing, just reusing some of the text from this PR, because it's not obvious what this is doing particularly if you're not familiar with `distinctUntilChanged` and such:

`// only create new state updates when the channel balance of ETH changes`